cmd/internal/get/graph.go

```diff
@@ -293,6 +342,26 @@ func printGraphByTarget(graph *api_schema.GraphResponse) {
 		}
 		return false
 	})
+	sort.Slice(graph.TLSEdges, func(i, j int) bool {
+		edgeI, edgeJ := graph.TLSEdges[i], graph.TLSEdges[j]
+		if edgeI.Source != edgeJ.Source {
+			return edgeI.Source < edgeJ.Source
+		}
+		if edgeI.Target != edgeJ.Target {
+			return edgeI.Target < edgeJ.Target
+		}
```

Comment:
Because this is printGraphByTarget, shouldn't target be the first key to sort on?

Reply:
Good catch. Fixed in aa1744b.